Source/Mosa.DeviceSystem/PCIGenericHostBridgeController.cs

```diff
@@ -3,28 +3,30 @@
 using Mosa.DeviceSystem;
 using Mosa.DeviceSystem.PCI;
 
-namespace Mosa.DeviceDriver.PCI.Intel
+namespace Mosa.DeviceSystem.PCI
 {
 	/// <summary>
 	/// Generic PCI Host Bridge Controller
 	/// </summary>
 	//[PCIDeviceDriver(ClassCode = 0x06, SubClassCode = 0x00, Platforms = PlatformArchitecture.X86AndX64)]
 	public class PCIGenericHostBridgeController : BaseDeviceDriver, IHostBridgeController
 	{
-		protected int ResetAddress { get; set; }
-		protected int ResetValue { get; set; }
+		protected ushort ResetAddress { get; set; }
+		protected byte ResetValue { get; set; }
 
 		public override void Initialize()
 		{
 			Device.Name = "PCIGenericHostBridgeController";
 		}
 
-		bool IHostBridgeController.CPUReset()
+		public bool CPUReset()
```

Comment:
Use the interface declaration.

Reply:
The problem then is that the method isn't public, so I can't access it from PCService.